src/Extensions/Compatability/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Omex.Extensions.Compatability
+{
+	/// <summary>
+	/// Class to register dependencies for Compatability classes
+	/// </summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>
+		/// Initialize deprecated static classes like Code and ULSLogger
+		/// </summary>
```

Comment:
Missing `<param>` and `<returns>`

Reply:
I don't think it will add any useful information here, this is an extension method on top of `IHostBuilder` so it practically doesn't have any parameters and it will return the same `IHostBuilder` for chaining,